### DISCUSSION TITLE
Закрыть всплывающие окна кликом вне окна

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#519] [ОШБК!] [answered]
Здравствуйте, заметили баг
На данный момент есть проблемка с длинными названиями, из-за которых не удаётся закрыть окно. было бы удобно закрывать окна iframe при помощи клика вне этого окна, либо по Esc.

![Screenshot_10](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/b705f685-8cad-486e-b4c7-9272faac1a60)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да, на это не рассчитывали, но поправят это не скоро.

Закрытие по блер точно не будет в стандарте, подумаем про параметр открытия - тоже небыстро.

**Reply by mihail-cornev** [reply]
Спасибо за обратную связь.
